docs/source/manual/configuration.rst

```diff
@@ -603,6 +603,10 @@ Name                   Default      Description
 ====================== ===========  ===========
 type                   REQUIRED     The appender type. Must be ``console``.
 threshold              ALL          The lowest level of events to print to the console.
+queueSize              256          The maximum capacity of the blocking queue.
+discardingThreshold    51           By default, when the blocking queue has 20% capacity remaining,
```

Comment:
I had to dig into Logback source code to figure out where 51 comes from. Let's call out that if no discarding threshold is specified, then a default of `queueSize / 5` is used.